dbt/graph/selector.py

```diff
@@ -4,10 +4,17 @@
 from dbt.utils import is_enabled, get_materialization, coalesce
 from dbt.node_types import NodeType
 from dbt.contracts.graph.parsed import ParsedNode
+import dbt.exceptions
 
 SELECTOR_PARENTS = '+'
 SELECTOR_CHILDREN = '+'
 SELECTOR_GLOB = '*'
+SELECTOR_DELIMITER = ':'
+
+
+class SELECTOR_FILTERS:
```

Comment:
This should derive from `object`, otherwise in 2.7 it'll act like an "old-style" class and potentially be a pain.

We could consider using this, if we really wanted enum support: https://pypi.org/project/enum34/

Reply:
Is there any benefit to using an enum over a class? Or I guess: do you have a preference?

Reply:
Well, that enum package is in the stdlib in 3.4+: https://docs.python.org/3.4/library/enum.html.

They just add a little syntactic sugar, and they can be nice for debugging.